acnh.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE user (
```

Comment:
Personal pet-peeve/nitpick that I'm not sure even matters: 
Change this to `CREATE TABLE IF NOT EXISTS` for idempotency.